### DISCUSSION TITLE
Lesson 8: QR Code Shows only the IPFS Id

### DISCUSSION THREAD

**Discussion by alymurtazamemon** [Cyfrin/foundry-full-course-cu#27] [Q&A] [answered]
This lessons show only single QR code and after scan it gives this;
```
ipfs://QmUgc1xpNgTMwSF2GBS5FTFKmSdrmSnLJD5DoYZtcQ2n5X
```
Is it intentional or is it a typo?

@PatrickAlphaC

**Comment by cromewar** [comment, marked as answer]
@alymurtazamemon I think is intentional:

![image](https://github.com/Cyfrin/foundry-full-course-f23/assets/20199180/71942a54-d4b9-4f01-86df-3965c71f3225)


Try with this: https://bafybeic6ikyzr4k654mkwtkoarxy3chv7ghtn4hq4kf3zqqunw2rzsdvsi.ipfs.dweb.link/

**Reply by alymurtazamemon** [reply]
@cromewar Thanks, ya previously I was just loading due to that I ask. Thanks by the way!

We can open it the IPFS way; [https://ipfs.io/ipfs/QmUgc1xpNgTMwSF2GBS5FTFKmSdrmSnLJD5DoYZtcQ2n5X/](https://ipfs.io/ipfs/QmUgc1xpNgTMwSF2GBS5FTFKmSdrmSnLJD5DoYZtcQ2n5X/)

**Reply by cromewar** [reply]
You are welcome!

**Reply by PatrickAlphaC** [reply]
Nice work you two!
